<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWin</class>
 <widget class="QMainWindow" name="HistoryWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>248</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>800</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="chButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>淬火</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hhButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>回火</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InfoBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>1101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
